### DISCUSSION TITLE
ImportError: No module named flask (Capstone Project Flask UI Dashboard)

### DISCUSSION THREAD

**Discussion by angeliainge** [teamalgoritma/community#193] [Q&A] [answered]
Halo,

Saya mengalami kesulitan saat menjalankan "python app.py" di Terminal VS Code. Di setiap kali saya ketikkan, selalu muncul tulisan :

Traceback (most recent call last):
  File "app.py", line 1, in <module>
    from flask import Flask, render_template
ImportError: No module named flask

Apakah bisa tolong dibantu?
Untuk library, sudah saya crosscheck ada dan sesuai semua dengan yang ada di requirements.txt

FYR saya menggunakan Mac OS 11.6.4

Attached untuk screenshot nya.. Thank you

<img width="598" alt="Screenshot 2022-07-10 at 21 08 40" src="https://user-images.githubusercontent.com/107099548/178148927-3568cc44-b9aa-4aaf-94e7-301f577d48b3.png">
<img width="1680" alt="Screenshot 2022-07-10 at 21 18 35" src="https://user-images.githubusercontent.com/107099548/178148928-136c041b-f5c5-45f6-b377-c5440f1927d8.png">

**Comment by Litaa** [comment, marked as answer]
Dear Ibu Angelia Inge,

Untuk mengatasi masalah yang Ibu hadapi kami menyarankan beberapa cara, yaitu :
1. Melakukan uninstall Flask kemudian menginstallnya kembali dengan perintah : 
- uninstall : `pip uninstall flask`
- install kembali : `pip install flask`
Kemudian lakukan restart pada IDE (visual studio code) anda
2. Membuat kembali environment baru dan kemudian menginstall kembali library2 yang dibutuhkan

Semoga kedua cara tersebut dapat membantu Ibu :). Terima kasih

**Reply by angeliainge** [reply]
Hai Kak @Litaa 

Maaf lupa update, untuk Flask nya sudah bisa jalan :)

Thank you feedback nya, untuk cara nya sudah saya lakukan semalam (uninstall dan install ulang, restart IDE, dan buat environment baru) namun tetap bisa.

Semalam saya coba langsung melalui terminal Mac, jadi tidak lewat terminal di VS Code dan bisa jalan.

thank you :)

**Reply by Litaa** [reply]
Baik sama-sama Ibu @angeliainge :)
Good luck untuk capstone nya ~
Saya mohon bantuannya untuk menandai jawaban sebagai "mark as answered" ya bu. Terima kasih
